### PR TITLE
[CL-389] Add end slot to non-text bit label content

### PR DESCRIPTION
**Pull request by vleague2**
## 🎟️ Tracking

[CL-389](https://bitwarden.atlassian.net/browse/CL-389)

## 📔 Objective

We want to update how the `bit-label` directive works, as part of the extension refresh UI updates. We'll be adding an `end` slot for content that should not get truncated (in the new label design, label text will be truncated in form fields if it is too long). To keep the changeset smaller (aka less teams involved) in the extension refresh feature branch, we are updating existing labels here in `main` by adding the `end` slot property. This will have no immediate user-facing changes or functionality changes in `main`. When the extension refresh branch is merged into `main`, these labels will then receive the updated styling.

A few links were also missing the `bitLink` component selector, so I added those as well.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[CL-389]: https://bitwarden.atlassian.net/browse/CL-389?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: bitwarden_license/bit-web/src/app/auth/sso/sso.component.html [hunk 1]
```diff
@@ -53,7 +53,7 @@
           id="memberDecryptionKey"
           [value]="memberDecryptionType.KeyConnector"
           [disabled]="!organization.useKeyConnector || null"
-          *ngIf="showKeyConnectorOptions"
+          *ngIf="true"
```
Comment (willmartian) [inline bitwarden_license/bit-web/src/app/auth/sso/sso.component.html:56]: This was left in on accident?

**Review by willmartian — COMMENTED** [review]
I think we are missing once instance here: https://github.com/bitwarden/clients/blob/0f375c3a0e848a555ae39ae8cabf38b6d49c6f53/libs/components/src/stories/kitchen-sink/components/kitchen-sink-form.component.ts#L74-L81

**Review by willmartian — DISMISSED** [review]
first

**Review by shane-melton — DISMISSED** [review]

**Review by djsmith85 — DISMISSED** [review]

**Review by r-tome — DISMISSED** [review]

**Review by jlf0dev — DISMISSED** [review]

**Review by amorask-bitwarden — APPROVED** [review]

**Comment by vleague2** [comment]
Sorry y'all, had to solve a merge conflict!

**Review by shane-melton — APPROVED** [review]

**Review by djsmith85 — APPROVED** [review]

**Review by r-tome — APPROVED** [review]

**Review by shane-melton — APPROVED** [review]

**Review by jlf0dev — APPROVED** [review]

**Review by willmartian — APPROVED** [review]

**Review by amorask-bitwarden — APPROVED** [review]

**Review by JaredSnider-Bitwarden — APPROVED** [review]

**Comment by vleague2** [comment]
Failing tests are fixed by #10512 in main, just haven't pulled in latest due to the number of reviewers this PR has.

**Pull request merged**